test-projects/gleam_gleam/.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: test
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
We don't have a `master` branch on this repo.

Reply:
that's a file generated by default by `gleam new`. I can remove this to avoid confusion